### ISSUE TITLE
Can I store an Object into Session?

### ISSUE THREAD

**Issue by sio4** [gobuffalo/buffalo#433]
Hello, I tried to store an object into Session. the code is, in brief:

```
c.Session().Set("current_user", user)
```

but it give me a error: "type not registered for interface: models.User"

Can I store entire object as is to Session?

**Comment by markbates** [comment]
I would never recommend storing complex types in a session. But if you want to you need to register the type. http://www.gorillatoolkit.org/pkg/sessions

**Comment by sio4** [comment]
where is the point to insert `gob.Register(&Person{})` in bufallo generated code?

Originally what I want is, storing a current users information on session and using it on template like `<%= user.name %>`.

I tried:
- storing whole `User` objecct on session and `context.Set("user", user)` on another middleware.
- storing `user.String()` (to get json text) on session and trying to use it on template.

Can I found any solution?
